boards/dactyl_manuform_carbonfet/README.md

```diff
@@ -0,0 +1,59 @@
+# Dactyl ManuForm Carbonfet
+
+The [Dactyl ManuForm Carbonfet](https://github.com/carbonfet/dactyl-manuform) is a handwired, split bodied, tented, curved key-well keyboard.  
+Forked from the [Dactyl ManuForm Mini](/boards/dactyl_manuform_mini), the Dactyl ManuForm Carbonfet appends the 5 key thumb cluster of the former with an additional key into a 2 row by 3 column arrangement.
+
+![White](https://i.imgur.com/0ugz1C9.jpg)  
+*Dactyl Manuform Carbonfet 5x6 variant*
+
+## Variants
+
+Dactyl Manuform Carbonfet's are built in variations that cater for different row and column counts.  
+Variants are denoted as `RowCount`*x*`ColumnCount` and share the common configuration of:
+- The finger key-well bottom row has 2 keys; 1 each in ring and middle columns.
+    - Exception to this rule is the `5x7` variant that has two additional keys in this row.
+- The thumb cluster has 5 keys: 2 (row) x 3 (column) arrangement with bottom left position ommited for left half, this arrangment mirrored for right half.
+
+### Row
+| Count | Description |
+| --- | --- |
+| 4 | Three rows, typically for alphabet and some punctuation characters, with two keys in bottom row of finger key-well section |
+| 5 | As *4 rows* with number row above |
+| 6 | As *5 rows* with function row above |
+
+### Column
+| Count | Description |
+| --- | --- |
+| 5 | A column for each finger with additional index finger column |  
+| 6 | As *5 columns* with additional pinky finger column |
+| 7 | As *6 columns* with either an additional index finger column (`5x7`) or additional pinky column (`6x7`) |
```

Comment:
Please align the column separators. (pad fields with spaces)

Reply:
amended as of commit 082ba79